### ISSUE TITLE
Zigbee2MQTT Ingress not working

### ISSUE THREAD

**Issue by PluginExcecption** [brenner-tobias/addon-cloudflared#637]
### The problem

When I try to open the Z2MQTT ingress it just shows me blank page and this error message in the DevTools console:

Uncaught SyntaxError: Invalid or unexpected token (at index-856ac473.js:1:11982)

I don't know if this is an issue on the side of Z2MQTT or the Cloudflared-Addon because it just stopped working after an update and the ingress is working locally, but not through the cloudflare tunnel.

### What version of Cloudflared has the issue?

5.1.9

### What type of installation are you running?

Home Assistant OS

### Add-on YAML Configuration

```yaml
external_hostname: ha.XXX.XX
additional_hosts: []
```

### Steps to reproduce the issue

1. Install Zigbee2MQTT
2. Install Cloudflared-Addon
3. Set up Cloudflared-Addon
4. Try to open the Z2MQTT ingress through the tunnel

**Comment by brenner-tobias** [comment]
Thanks for raising this. Just to clarify: Did this work previously and if so, what did you change / update and from which versions?

**Comment by elcajon** [comment]
> I don't know if this is an issue on the side of Z2MQTT or the Cloudflared-Addon because it just stopped working after an update and the ingress is working locally, but not through the cloudflare tunnel.

FYI, I am currently using the latest Cloudflare edge version and zigbee add-on version 1.37.1-1 without any problems.

**Comment by PluginExcecption** [comment]
I've tried it on multiple devices, so caching shouldn't be the issue. The other ingresses like VS Code work via Cloudflared, only Z2MQTT doesn't. I've changed nothing manually and only applied the latest updates of the Cloudflared- and the Z2MQTT-Addons.
My Z2MQTT-Addon is also on version 1.37.1-1 and the Front-End there is set to the default settings.

I updated Z2MQTT from 1.37.0 to 1.37.1 around 4 days ago and last week from 1.36.1 to 1.37.1. The last time I updated cloudflared was on the 26th of April from 5.1.7 to 5.1.9.
